chains/evm/.solhint-test.json

```diff
@@ -4,36 +4,36 @@
   "rules": {
     "compiler-version": ["off", "^0.8.0"],
     "const-name-snakecase": "off",
-    "constructor-syntax": "error",
```

Comment:
This file is re-ordered to first list all the items that are turned off, and then the rules that we turn on. A few items are added due to them being default in 6.0 and some are removed as they are either no longer required (reason-string) or default (interface-starts-with-i).